tests/e2e/controller_test.go

```diff
@@ -153,7 +150,7 @@ func NewTestContext() (*testContext, error) {
 		ctx:                   context.TODO(),
 		testDsc:               testDSC,
 		testDSCI:              testDSCI,
-		platform:              release.Name,
+		platform:              testDSCI.Status.Release.Name,
```

Comment:
`setupDSCICR` only creates the CR struct and sets up the Spec field, therefore, `testDSCI.Status.Release.Name `will be empty.